resources/dicts/patrols/general/medcat.json

```diff
@@ -81,13 +82,14 @@
     "success_text": [
         "They have an in depth conversation about r_c's vision as they walk. By the time they head back to camp, they have a good idea of what they should do with the information.",
         "p_l takes r_c aside, giving them the full benefits of p_l's seniority and experience as they discuss r_c's vision. As a medicine cat, they don't take the responsibility lightly, and both of them work together to divine the vision's meaning.",
-        "s_c listens intently and tells r_c not to be weirded out, but they had seen the dream, too. r_c is alarmed but s_c soothes them and explains that StarClan sometimes shows them others' dreams, but this one luckily was just a mild warning."
+        "s_c listens intently and tells r_c not to be weirded out, but they had seen the dream, too. r_c is alarmed, but s_c soothes them, explaining that StarClan sometimes shows them others' dreams. This one, luckily, was nothing but a mild warning."
     ],
     "fail_text": [
         "p_l shakes their head sadly. They can't make sense of the vision.",
-        "r_c notices p_l looking at them enviously, and withdraws from the group, dismissing the dream as unimportant."
+        "s_c notices p_l looking at them enviously, and withdraws from the group, dismissing the dream as unimportant."
```

Comment:
This is another case of fail and success being inconsistent on r_c or p_l being s_c.  It also lacks any "rc_has_stat" tags